src/main/java/com/emeq/counter/HelloController.java

```diff
@@ -0,0 +1,17 @@
+package com.emeq.counter;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HelloController {
+	
+	private static final String template = "Hello, %s!";
+
+	@GetMapping("/hello")
+	public Hello hello(@RequestParam(value = "name", defaultValue = "World") String name) {
+		return new Hello(String.format(template, name));
```

Comment:
According to accepted criteria we wait to function return as `{hello: "world"}`.